### ISSUE TITLE
Refactor get_content controller

### ISSUE THREAD

**Issue by shouston3** [LDMW/app#287]
As of #285 the current signature is [a bit messy](https://github.com/LDMW/app/pull/285/files#diff-cedc8ca18b8e918d10bfed845522f66fR37).
And the function is [quite inflexible and inefficient](https://github.com/LDMW/app/pull/285/files#diff-7ea04a895881487e0d2273998c808f1eR43)
You can only pass in one atom for it to fetch.
We typically would like to get several bits of content at once and currently we have to make numerous queries when it should only be one.

The proposed signature would be something like:
```elixir
iex>R.get_content(:alphatext, "feedback")
"<h1>feedback alphatext content</h1>"
iex>R.get_content(:alphatext)
"<h1>homepage alphatext content</h1>"
iex>R.get_content([:alphatext, :alpha], "feedback")
["<h1>feedback alphatext content</h1>", "<h1>feedback alpha content</h1>"]
iex>R.get_content([:alphatext, :alpha, :body], "home")
["<h1>feedback alphatext content</h1>", "<h1>feedback alpha content</h1>", "<h1>homepage body content</h1>"]
```

**Comment by shouston3** [comment]
Put on hold until this has been answered:

https://stackoverflow.com/questions/44390950/dynamically-selecting-map-ecto

**Comment by shouston3** [comment]
Fixed by #299
